src/doc/using_visit/Operators/OperatorTypes/ExtrudeStacked_operator.rst

```diff
@@ -0,0 +1,87 @@
+.. _ExtrudeStacked operator:
+
+ExtrudeStacked operator
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The ExtrudeStacked operator uses a scalar field on a 2D mesh to
+extrude each node in the input mesh, resulting in a 3D mesh. The The
+ExtrudeStacked operator can also extrude plots whose input data was
+produced from higher dimensional data that has been
+sliced. Furthermore, the ExtrudeStacked operator allows you to display
+multiple scalar fields in a single plot such as when a Pseudocolor
+plot of scalar variable A is extrudes by scalar variable B (see:
+:numref:`Figure %s <extrude>`).
+
+.. _extrude:
+
+.. figure:: images/extrude.png
+
+   ExtrudeStacked operator example: A simple structured grid extruded
+   by the cell number.
+
+Using the ExtrudeStacked operator
+"""""""""""""""""""""""""""""""""
+
+The ExtrudeStacked operator can be used to create 3D height field
+plots by applying operator to a plot that accepts scalar values. The
+ExtrudeStacked operator is quite flexible because it can extrude by
+more than one variable and coloring by the same variable.
+
+Since the ExtrudeStacked operator uses a scalar variable to extrude
+all of the points in the mesh, the ExtrudeStacked operator has a
+number of controls related to scaling scalar data. For example, the
+ExtrudeStacked operator allows you to artificially set minimum and
+maximum values for the scalar variable so you can eliminate data that
+might otherwise cause your extruded plot to be stretched undesirably
+in the Z direction. To set minimum and maximum values for the
+ExtrudeStacked operator, set the **Min** or **Max** check boxes in the
+**ExtrudeStacked attributes window** (see :numref:`Figure %s
+<extrudeWindow>`) and type new values into the adjacent text
+fields.
```

Comment:
```suggestion
in the Z direction. To set minimum and maximum values for a variable, make sure that variable is highlighted in the table, then enter the desired minimum or maximum in the **Min** or **Max** text edit boxes below the table. (see :numref:`Figure %s
<extrudeWindow>`)  The table entries will be updated to the values entered.
```

The Min/Max aren't check boxes.
Also, I think it would be better if the example figure had multiple variables listed in the table, and min or max changed for the hightlighted variable to show that entering the min/max updates what is shown in the table, too.

Reply:
As I had an example with a scaled mesh I used that as an example for window.